src/screens/student/grades/index.tsx

```diff
@@ -14,7 +17,21 @@ export const GradesContent = () => {
         flexDirection: "row",
       }}
     >
+        <Card
+            title={"Grades: "}
+            hoverable
+            style={{height: 400}}
+        >
       <GradesTable data={data} />
+        </Card>
+
+        <Card
+            title={"Optional courses: "}
+            hoverable
+            style={{height: 400, marginLeft: 100}}
```

Comment:
it's better to give the component a class instead of adding inline style for every elements, it's cleaner :D